integration-tests/features/environment.py

```diff
@@ -284,6 +288,14 @@ def _send_json_file(endpoint, filename):
     return response
 
 
+def _parse_int_env_var(env_var_name):
+    val = os.environ.get(env_var_name)
+    try:
+        return int(val)
+    except (TypeError, ValueError):
```

Comment:
Probably keeping this exception would make errors based on env variables more visible when debugging typos.